src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -503,7 +505,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
                 // 2. GroupNormalizationDecomposition produce MVN, and MVN have a conditional pass MVN6Decomposition. If call MVN6Decomposition again after
                 //    snippets pipeline as well, where MVN is decomposed to simple ops, these simple ops will not tokenized into subgraph again.
                 // CVS-134277 to fully enable GN as snippets to disable this GroupNormalizationDecomposition entirly.
-                if (node->is_dynamic() || inferencePrecision != element::f32)
+                if (node->is_dynamic() || !one_of(inferencePrecision, element::f32, element::undefined))
                     return false;
                 const auto group_norm = ov::as_type_ptr<const ov::op::v12::GroupNormalization>(node);
                 if (!group_norm)
```

Comment:
At the moment Snippets support only FP32 `GN` op. In these cases `ov::pass::GroupNormalizationDecomposition` should be passes to allow Snippets tokenize `GN` to Subgraph.
If `inferencePrecision == element::undefined`, we should check that `GN` is `FP32`:
```suggestion
                if (node->is_dynamic() || !one_of(inferencePrecision, element::f32, element::undefined))
                    return false;
                const auto group_norm = ov::as_type_ptr<const ov::op::v12::GroupNormalization>(node);
                if (!group_norm || !implication(inferencePrecision == element::undefined, group_norm->get_element_type() == f32))
```

Reply:
Done